tests/basic-test.sh

```diff
@@ -541,6 +541,24 @@ fi
 assert_file_has_mode checkout-test2-override/a/readable-only 600
 echo "ok commit statoverride"
 
+cd ${test_tmpdir}
+rm test2-checkout -rf
+$OSTREE checkout test2 test2-checkout
+cd test2-checkout
+touch user-wx
+chmod 0755 user-wx
+touch group-wx
+chmod 0575 group-wx
+touch both-wx
+chmod 0775 both-wx
```

Comment:
Maybe a sanity-check that a writable but not executable file doesn't get its write perms stripped?